packages/eslint-plugin/package.json

```diff
@@ -103,7 +103,7 @@
   },
   "peerDependencies": {
     "@typescript-eslint/parser": "^7.0.0",
-    "eslint": "^8.57.0"
+    "eslint": "^9.0.0"
```

Comment:
This should include v8 and v9 shouldn't it?

Reply:
Right! Yes, I'd meant to ask to confirm. This is better. +1